blockdevice/stats_test.go

```diff
@@ -104,4 +104,116 @@ func TestBlockDevice(t *testing.T) {
 	if device1stats.DiscardTicks != 12 {
 		t.Errorf(failMsgFormat, "Incorrect discard ticks", 12, device1stats.DiscardTicks)
 	}
+	blockQueueStat, err := blockdevice.SysBlockDeviceQueueStats(devices[1])
```

Comment:
Ideally we should use a loop here as well or just a expected struct and use DeepEqual or somtehing.

Reply:
yep, DeepEqual sound like a good idea.